src/main/resources/db/changelog/changes/09-insert-books.yaml

```
@@ -0,0 +1,68 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert_books
+      author: alex
```

```suggestion
      author: alex_liubchenko
```
it's better to pass also last name